driver_test.go

```diff
@@ -45,14 +42,33 @@ type Connector struct {
 
 func NewConnector() (*Connector, error) {
 
-	// Configure emulator if set.
-	spannerHost, ok := os.LookupEnv("SPANNER_EMULATOR_HOST")
-
 	ctx := context.Background()
 
+	adminClient, err := CreateAdminClient(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	dataClient, err := spanner.NewClient(ctx, dsn)
+	if err != nil {
+		return nil, err
+	}
+
+	curs := &Connector{
```

Comment:
nit: The struct says it's a connector but the variable name says it's a cursor. Since this is temporary until DDL is working, this is fine as it is. Just something to consider.

Reply:
Did a quick search n replace